annotation/tests/test_jobs.py

```diff
@@ -0,0 +1,139 @@
+import uuid
+from unittest.mock import MagicMock
+
+import pytest
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import check_annotators, update_inner_job_status
+from annotation.models import Category, File, Job, User
+from annotation.schemas import (
+    CategoryTypeSchema,
+    FileStatusEnumSchema,
+    JobStatusEnumSchema,
+    ValidationSchema,
+)
+
+
+@pytest.fixture
+def job_tenant():
+    return "test"
+
+
+@pytest.fixture
+def job_ids():
+    return (1, 2, 3)
+
+
+ANNOTATORS = (
+    User(user_id="82533770-a99e-4873-8b23-6bbda86b59ae"),
+    User(user_id="ef81a4d0-cc01-447b-9025-a70ed441672d"),
+)
+
+
+@pytest.fixture
+def job_annotators():
+    return (
+        User(user_id="82533770-a99e-4873-8b23-6bbda86b59ae"),
+        User(user_id="ef81a4d0-cc01-447b-9025-a70ed441672d"),
+    )
```

Comment:
same data, remove constant please

Reply:
I can't use in function "test_check_annotators" parametrize the fixture "job_annotators" that's why I have the constant one.

Reply:
The reply to previous conversation didn't load for me. I just saw the reply there.